src/routes/following.route.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from "express";
+
+import { isAuthorized } from "../middlewares/isAuthorized.middlewares";
+import {
+  getFollowing,
+  addFollowing,
+  deleteFollowing,
+} from "../services/following.service";
+
+const followingRouter = express.Router();
+
+followingRouter
+  .route("/")
+  .get(isAuthorized, async (req: Request, res: Response) => {
+    const result = await getFollowing(req.query);
+    res.send(result);
+  });
+
+followingRouter
+  .route("/")
+  .post(isAuthorized, async (req: Request, res: Response) => {
+    await addFollowing(req.body);
+    res.sendStatus(200);
```

Comment:
```suggestion
    const newFollowing = await addFollowing(req.body);
    res.status(201).send(newFollowing);
```
we need to return created object